hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBCheckpointUtils.java

```diff
@@ -67,4 +68,15 @@ public static boolean waitForCheckpointDirectoryExist(File file)
       return false;
     }
   }
+
+  /**
+   * Wait for checkpoint directory to be created for 5 secs with 100 millis
+   * poll interval.
+   * @param file Checkpoint directory.
+   * @return true if found.
+   */
+  public static boolean waitForCheckpointDirectoryExist(File file)
```

Comment:
```suggestion
  @VisibleForTesting
  public static boolean waitForCheckpointDirectoryExist(File file)
```

Reply:
This is use by [bootstrapping code](https://github.com/apache/ozone/blob/6113470e667f31955d4a044a56871c21bbe591de/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java#L247). Not sure if we need to change `waitForCheckpointDirectoryExist(File file)` to `waitForCheckpointDirectoryExist(File file, Duration timeout)` in bootstrapping?

Reply:
yeah checked the usage. We should be good here. It throws an IOException in the case the directory doesn't exist in the bootstrapping code. It would retry once again if it fails.